2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,20 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
+
     // TODO
+//  all elements of article titles array
+  
+     let sortedArticleTitles = []
+  for (let i = 0; i < allArticleTitles.length; i++ ){
+    if (allArticleTitles[i].length <= 65 ){
+      sortedArticleTitles.push(allArticleTitles[i])
+    } else if (allArticleTitles[i].length === 0 ){
+      sortedArticleTitles = []
+    }
```

Comment:
no need for this `else if` statement here - are you trying to put a check in for allArticleTitles.length being 0? If so the loop would not run as `i < allArticleTitles.length` would equate to `i < 0` and `sortedArticleTitles` would get returned as an empty array :)

Reply:
Hi, yeah I was trying to check for an empty array. So if the input is an empty array then the return should be empty. Now it makes sense that I should not put else if. thank u so much.  I am updating my solution.